### ISSUE TITLE
`eo-runtime` test coverage

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/eo#1614]
We experience some problems with merging new PR`s since test coverage of `eo-runtime` doesn't match `jacoco` checks:
```java
[ERROR] Failed to execute goal org.jacoco:jacoco-maven-plugin:0.8.8:check (jacoco-check) on project eo-runtime: Coverage checks have not been met. See log for details. 
```
Warnings:
```java
[WARNING] Rule violated for bundle eo-runtime: instructions covered ratio is 0.36, but expected minimum is 0.40
[WARNING] Rule violated for bundle eo-runtime: lines covered ratio is 0.38, but expected minimum is 0.42
[WARNING] Rule violated for bundle eo-runtime: complexity covered ratio is 0.51, but expected minimum is 0.53
[WARNING] Rule violated for bundle eo-runtime: methods covered ratio is 0.51, but expected minimum is 0.53
[WARNING] Rule violated for bundle eo-runtime: classes missed count is 155, but expected maximum is 145
```

The issue have to be solved as soon as possible, since it blocks all other PR`s

**Comment by yegor256** [comment]
@volodya-lombrozo please, solve it if you can

**Comment by 0pdd** [comment]
@volodya-lombrozo 4 puzzles [#1616](https://github.com/objectionary/eo/issues/1616), [#1617](https://github.com/objectionary/eo/issues/1617), [#1618](https://github.com/objectionary/eo/issues/1618), [#1619](https://github.com/objectionary/eo/issues/1619) are still not solved.

**Comment by 0pdd** [comment]
@volodya-lombrozo 3 puzzles [#1616](https://github.com/objectionary/eo/issues/1616), [#1617](https://github.com/objectionary/eo/issues/1617), [#1618](https://github.com/objectionary/eo/issues/1618) are still not solved; solved: [#1619](https://github.com/objectionary/eo/issues/1619).

**Comment by 0pdd** [comment]
@volodya-lombrozo 2 puzzles [#1616](https://github.com/objectionary/eo/issues/1616), [#1617](https://github.com/objectionary/eo/issues/1617) are still not solved; solved: [#1618](https://github.com/objectionary/eo/issues/1618), [#1619](https://github.com/objectionary/eo/issues/1619).

**Comment by 0pdd** [comment]
@volodya-lombrozo 2 puzzles [#1616](https://github.com/objectionary/eo/issues/1616), [#1664](https://github.com/objectionary/eo/issues/1664) are still not solved; solved: [#1617](https://github.com/objectionary/eo/issues/1617), [#1618](https://github.com/objectionary/eo/issues/1618), [#1619](https://github.com/objectionary/eo/issues/1619).

**Comment by 0pdd** [comment]
@volodya-lombrozo the puzzle [#1616](https://github.com/objectionary/eo/issues/1616) is still not solved; solved: [#1617](https://github.com/objectionary/eo/issues/1617), [#1618](https://github.com/objectionary/eo/issues/1618), [#1619](https://github.com/objectionary/eo/issues/1619), [#1664](https://github.com/objectionary/eo/issues/1664).
